app/api/routes.py

```diff
@@ -688,7 +688,7 @@
     TicketDetail,
     'ticket_detail',
     '/tickets/<int:id>',
-    '/attendees/<int:attendee_id>/ticket',
+    '/attendees/<int:attendee_id>/ticket'
```

Comment:
Please don't include unrelated changes in a PR. 
Final `,` are a good thing, since it keeps diffs small.
Please adjust your `black` configuration (or whatever formatter you are using)